lua/nvim-tree/explorer/reload.lua

```diff
@@ -162,7 +162,7 @@ function M.reload(node, git_status)
     return ns
   end
 
-  sorters.sort(node.nodes)
+  M.sorters:sort(node.nodes)
```

Comment:
Similar lifecycle issue with `Explorer`.

This one is problematic as reload is not a class. The functionality should go into `Explorer` however that's out of scope here.

For now we can do something like:
```lua
  local explorer = require("nvim-tree.core").get_explorer()
  if explorer then
    explorer.sorters:sort(node.nodes)
  end
```

Reply:
Raised #2837